crypto-rng.cabal

```diff
@@ -14,7 +14,8 @@ maintainer:          Mikhail Glushenkov <mikhail.glushenkov@scrive.com>,
 copyright:           Scrive AB
 category:            Crypto
 build-type:          Simple
-tested-with:         GHC == 8.0.2, GHC == 8.2.2, GHC == 8.4.4, GHC == 8.6.2
+tested-with:         GHC == 8.0.2, GHC == 8.2.2, GHC == 8.4.4
```

Comment:
Can you make this be `GHC ==X || ==Y || ...` instead (like in hpqtypes)? Then there is no need to break lines.